app/javascript/components/Alerts.js

```diff
@@ -0,0 +1,10 @@
+const ERROR_MESSAGE = "Oops! Looks like something went wrong. Please try again."
+
+const info = (message) => alert(message)
+
+const error = () => alert(ERROR_MESSAGE)
```

Comment:
@fddayan I added a standard error alert using the same default error message we use on the web app for consistency.

Reply:
👍